Integrations/VMRay/VMRay.yml

```diff
@@ -519,9 +519,23 @@ script:
     description: Retrieves a sample using the sample ID. (Deprecated)
     execution: false
     name: upload_sample
+  - arguments:
+    - default: true
+      description: A CSV list of hashes of the file to query. Supports MD5, SHA1,
+        and SHA256.
+      isArray: false
+      name: file
+      required: true
+      secret: false
+    deprecated: false
+    description: Checks the file reputation of the specified hash.
+    execution: false
+    name: vmray-check-file
   dockerimage: demisto/python
   isfetch: false
+  longRunning: false
+  longRunningPort: false
   runonce: false
-  script: ''
+  script: '-'
   type: python
-fromversion: 4.0.0
+sourcemoduleid: vmray
```

Comment:
why did you add this?

Reply:
I didn't edit the original code as I couldn't import it to test, so had to copy the original code into a "v2" integration.